README.md

```diff
@@ -238,6 +238,48 @@ But patch is welcome.
 * HTTP headers that start with 'Grpc-Metadata-' are mapped to gRPC metadata (prefixed with `grpcgateway-`)
 * While configurable, the default {un,}marshaling uses [jsonpb](https://godoc.org/github.com/golang/protobuf/jsonpb) with `OrigName: true`.
 
+# Generating for unannotated protos
```

Comment:
What would you think about making this a page in the `/docs` section and linking to it?

Reply:
Sgtm. Much better than littering the landing page.